composer.json

```diff
@@ -25,7 +25,7 @@
   ],
   "license": "GPL-2.0-or-later",
   "require": {
-    "php": ">= 7.2.0, <= 7.4.99",
+    "php": ">= 7.2.0, <= 7.4.99 | ^8.0",
```

Comment:
- The single pipe `|` is deprecated and in backward compatibility mode => use `||`
- `^8.0` would allow all versions up to 8.9.99, we should better stick to `>= 8.0.0, <= 8.1.99`. The comma itself is same as a blank space so we could remove it and all of this would thus become (sticking to official syntax on https://getcomposer.org/doc/articles/versions.md#version-range):

```
"php": ">=7.2.0 <=7.4.99 || >=8.0.0 <=8.1.99",
```